src/MiniCover/Instrumentation/CustomAssemblyResolver.cs

```diff
@@ -25,22 +46,37 @@ AssemblyDefinition GetAssembly(string file, ReaderParameters parameters)
 
         protected override AssemblyDefinition SearchDirectory(AssemblyNameReference name, IEnumerable<string> directories, ReaderParameters parameters)
         {
-            foreach (var directory in directories)
+            if (_dependencyContext != null)
```

Comment:
I would prefer to return null if _dependencyContext  is null.
Nice to add a log message for this case

Reply:
We still need to support the non .net core assemblies. They don't have dependency context.
That's why we need to fallback to base.SearchDirectory.